index.js

```diff
@@ -46,7 +59,11 @@ class LESSCompiler {
           }
           return reject(err);
         }
-        return resolve({data: output.css});
+        if (this.config.cssModules) {
+          return cssModulify(path, output.css).then(result => resolve(result), error => reject(error));
```

Comment:
Nope, we can't do that since the outermost promise was constructed with `new Promise`, hence we have to use its resolve/reject callbacks. We can however shorten `then(...)` to `then(resolve, reject)`